src/main/java/com/datatorrent/netlet/AbstractClient.java

```diff
@@ -38,6 +40,50 @@
 {
   private static final int THROWABLES_COLLECTION_SIZE = 4;
   public static final int MAX_SENDBUFFER_SIZE;
+  private static final SelectionKey invalidSelectionKey = new SelectionKey()
```

Comment:
I can make it public in the next minor release, so it will be accessible from outside. I can't do this in the patch release.